src/shared/components/QueryBuilder/domain/helpers/__tests__/queryToFilters.spec.ts

```diff
@@ -79,6 +79,28 @@ const cases: TestCase[] = [
       },
     ],
   ],
+  [
+    ' process_cpu:cpu:nanoseconds:cpu:nanoseconds{service_name="core-requests",free_text=~"one,two"}',
```

Comment:
Modified the code and added a unit test to account for values that could contains commas.